etl-api/src/configs/pipeline.rs

```diff
@@ -89,6 +94,7 @@ pub struct PartialApiPipelineConfig {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct StoredPipelineConfig {
     pub publication_name: String,
+    pub replication_slot: ReplicationSlotConfig,
```

Comment:
This will crash for existing stored configs.
```suggestion
    #[serde(default)]
    pub replication_slot: ReplicationSlotConfig,
```

This has to be paired with the other comment that I left.